shared/wallets/transaction/index.stories.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import * as React from 'react'
+import {Box, Text} from '../../common-adapters'
+import {storiesOf} from '../../stories/storybook'
+import Transaction from '.'
+
+const load = () => {
+  storiesOf('Wallets/Transaction', module)
+    .addDecorator(story => <Box style={{maxWidth: 520}}>{story()}</Box>)
+    .add('Default wallet to Keybase User', () => (
+      <Transaction
+        large={true}
+        timestamp={new Date()}
+        yourRole="sender"
+        counterparty="paul"
+        counterpartyType="keybaseUser"
+        amountUser="$12.50"
+        amountXLM="53.1688643 XLM"
+        note="Short note."
+      />
+    ))
+    .add('Keybase User to Default wallet', () => (
+      <Transaction
+        large={false}
+        timestamp={new Date()}
+        yourRole="receiver"
+        counterparty="paul"
+        counterpartyType="keybaseUser"
+        amountUser="$100"
+        amountXLM="42.535091 XLM"
+        note="Stellar deal!! You guys rock. This is to show a very long private note. Blah blah blah blah."
+      />
+    ))
+    .add('Default wallet to Stellar Public Key', () => (
+      <Transaction
+        large={true}
+        timestamp={new Date()}
+        yourRole="sender"
+        counterparty="G43289XXXXX34OPL"
+        counterpartyType="stellarPublicKey"
+        amountUser="$12.50"
+        amountXLM="53.1688643 XLM"
+        note="Short note."
+      />
+    ))
+    .add('Anonymous wallet to Keybase User', () => <Text type="BodyBig">TBD</Text>)
```

Comment:
What do anonymous wallets look like? is it just a Stellar public key?

Reply:
yes, anonymous is an incorrect term

Reply:
Fixed.